backend/apps/slack/admin/entity_channel.py

```
@@ -0,0 +1,14 @@
+from django.contrib import admin
+from django.contrib.contenttypes.admin import GenericTabularInline
+from ..models import EntityChannel
+
+@admin.action(description="Mark selected EntityChannels as reviewed")
+def mark_as_reviewed(modeladmin, request, queryset):
```

Ruff (0.12.2)

Missing docstring in public function

(D103)